README.md

```diff
@@ -232,7 +232,7 @@ Check solve section for steganography.
 - [SQLMap](https://github.com/sqlmapproject/sqlmap) - Automatic SQL injection and database takeover tooli
 - [W3af](https://github.com/andresriancho/w3af) -  Web Application Attack and Audit Framework.
 - [XSSer](http://xsser.sourceforge.net/) - Automated XSS testor
-
+- [Commix](https://github.com/commixproject/commix) - Automated All-in-One OS Command Injection and Exploitation Tool.
```

Comment:
Hi, Thanks for the PR. But following contributing guidelines of the repository, we need to put this at correct alphabetical position.